packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -123,6 +129,23 @@ export default class Sayt extends LitElement {
     }
   }
 
+  /**
+   * Determines whether an autocomplete call should be made based on minchars attribute.
+   * Then fires the autocomplete fetch data event to communicate with the sayt driver.
+   *
+   * @param event An event that can contain a searchbox ID.
+   */
+  requestSayt(event: CustomEvent) {
+    if (!this.isCorrectSayt(event)) return;
+    if(event.detail.value.length >= this.minchars) {
```

Comment:
Lines 139 and 140 could be combined:
```suggestion
    if (!this.isCorrectSayt(event) || !event.detail.value.length >= this.minchars) return;
```

And then you don't need the second if statement.